baselines/dom.generated.d.ts

```diff
@@ -14869,6 +14909,70 @@ declare var ImageData: {
     new(data: Uint8ClampedArray, sw: number, sh?: number, settings?: ImageDataSettings): ImageData;
 };
 
+/**
+ * Available only in secure contexts.
+ *
+ * [MDN Reference](https://developer.mozilla.org/docs/Web/API/ImageDecoder)
+ */
+interface ImageDecoder {
```

Comment:
This is blink only right now...

Reply:
MDN takes a few days after BCD release happens. Best to check BCD directly

Reply:
Same in the data. 
133 will be released 2024-11-26 as shown nicely in mdn :)

https://github.com/mdn/browser-compat-data/blob/main/api/ImageDecoder.json

Reply:
That's fine, any features riding release train as enabled-by-default are treated as a support.